hrjobcontract/js/test/services/utils.service.spec.js

```diff
@@ -13,6 +13,17 @@ define([
   describe('utilsService', function () {
     var $httpBackend, $provide, $rootScope, $q, $uibModal, utilsService, apiService,
       AbsencePeriod, $window;
+    var groupNames = 'hrjc_health_insurance_provider';
+    var optionValueApiParams = {
```

Comment:
both `groupNames` and `optionValueApiParams` are only used in `describe('getOptionValues')`, so declare it there, and assign their value in a `beforeEach`